pnpm-lock.yaml

```diff
@@ -63390,3 +63373,7 @@ packages:
     dependencies:
       commander: 2.20.3
     dev: false
+
+settings:
+  autoInstallPeers: false
+  excludeLinksFromLockfile: false
```

Comment:
Not sure what this change does, but it doesn't seem required, is it ?
If not can you please remove it? (by re-generating this file)

Reply:
it automatically got moved from top to bottom of the file and I tried regenerating and it is doing same. should I just revert those 3 line changes?

Reply:
Nevermind, I didn't see that it just got moved around in the file indeed, and besides this change seems to be on `develop` too already. All good :+1: